incubator/swift/image/Dockerfile-stack

```diff
@@ -1,10 +1,10 @@
-FROM swift:5.1
+FROM swift:5.2
 
 LABEL Description="Appsody runtime for Swift applications"
 LABEL maintainer="Ian Partridge <i.partridge@uk.ibm.com>, David Jones <djones6@uk.ibm.com>"
 
 RUN apt-get update \
- && apt-get install -y zlib1g-dev libcurl4-openssl-dev libssl-dev \
+ && apt-get install -y libcurl4-openssl-dev libssl-dev \
```

Comment:
Hmm actually @djones6 didn't we leave `zlib1g-dev` in on purpose because it's a dependency of SwiftNIO and we wanted the `swift` stack to work out-of-the-box with SwiftNIO?

Reply:
Ah, I removed it because I noticed it is now part of the base Docker image: https://github.com/apple/swift-docker/blob/5a513a780593e38126dbb09828e30167f132b599/5.2/ubuntu/18.04/Dockerfile#L16

Reply:
Oh cool, they must have added that. Thanks.